platform/podjs/tsconfig.json

```diff
@@ -16,4 +16,4 @@
   },
   "include": ["./src/**/*.ts"],
   "exclude": ["node_modules", "dist"]
-}
+}
```

Comment:
This is unrelated? Also, bound to change when loaded into some other IDE, I guess.